source/Octopus.Tentacle.Tests.Integration/Support/ClientAndTentacle.cs

```diff
@@ -42,15 +46,24 @@ public LegacyTentacleClientBuilder LegacyTentacleClientBuilder(AsyncHalibutFeatu
             TemporaryDirectory = temporaryDirectory;
             RpcRetrySettings = rpcRetrySettings;
             this.ServiceEndPoint = serviceEndPoint;
+            this.logger = logger.ForContext<ClientAndTentacle>();
         }
 
         public async ValueTask DisposeAsync()
         {
-            Server.Dispose();
-            PortForwarder?.Dispose();
-            await RunningTentacle.DisposeAsync();
+            logger.Information("Starting DisposeAsync");
+
+            logger.Information("Starting RunningTentacle.DisposeAsync and Server.Dispose and PortForwarder.Dispose");
```

Comment:
Do these in parallel to see if it can improve the speed of Halibut disposal e.g. by killing Prot Forwarder and RunningTentacel connections.

If Halibut still appears to be the source of failure then we may need to investigate net48 sync Halibut disposal

```
00:00:16.5808096 [ClientAndTentacle] Starting DisposeAsync
00:00:16.5808173 [ClientAndTentacle] Starting Server.Dispose
00:00:59.9038781 [ClientAndTentacle] Starting PortForwarder.Dispose
00:00:59.9038877 [PortForwarder] Start Dispose Pumps
00:00:59.9038944 [PortForwarder] Fisinshed Dispose Pumps
00:00:59.9039451 [ClientAndTentacle] Starting RunningTentacle.DisposeAsync
00:00:59.9039588 [RunningTentacle] Starting DisposeAsync
00:00:59.9039660 [RunningTentacle] Starting Stop
00:00:59.9549391 [RunningTentacle] Starting deleteInstanceFunction
00:01:00.0028101 [ClientFileTransferRetriesTimeout] Staring Test Tearing Down
00:01:00.0028286 [ClientFileTransferRetriesTimeout] Cancelling CancellationTokenSource
00:01:00.0028394 [ClientFileTransferRetriesTimeout] Disposing CancellationTokenSource
00:01:00.0028485 [ClientFileTransferRetriesTimeout] Finished Test Tearing Down
```